estate/models/estate_property.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+
+from odoo import api, fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+    name = fields.Char(string="Title")
+    seller_id = fields.Many2one(
+        "res.users", string="Salesperson", index=True, copy=True, default=lambda self: self.env.user
+    )
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        default=datetime.datetime.now()
+        + datetime.timedelta(days=90),  # Setting the date availability 3 months from now.
+        copy=False,
+    )
+    validity_days = fields.Integer(default=7)
+    deadline_date = fields.Date(compute="_compute_deadline_date", inverse="_inverse_validity_days")
+    expected_price = fields.Float()
+    selling_price = fields.Float(readonly=True, copyright=False, copy=False)
+    bedrooms = fields.Integer(default=4)
+    living_area = fields.Integer()
+    garden_area = fields.Integer()
+    total_area = fields.Integer(compute="_compute_total_area")
+    facades = fields.Integer()
+    has_garage = fields.Boolean()
+    has_garden = fields.Boolean()
+    active = fields.Boolean(default=True)
+    garden_orientation = fields.Selection(
+        selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")]
+    )
+    status = fields.Selection(
+        selection=[("new", "New"), ("offer_receieved", "Offer Recieved"), ("sold", "Sold"), ("cancelled", "Cancelled")],
```

Comment:
We use American English at Odoo (no need to edit the technical name now)
```suggestion
        selection=[("new", "New"), ("offer_receieved", "Offer Recieved"), ("sold", "Sold"), ("cancelled", "Canceled")],
```

Reply:
I didn't know that the convention was American English and I didn't know too that 'cancelled' might be with a single 'L' 😂.